nominees/viame.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "VIAME",
+  "description": "A DIY AI system for analyzing imagery and video, orginally targeting marine species analysis.",
+  "website": "https://www.viametoolkit.org/",
+  "license": [
+    {
+      "spdx": "BSD-3-Clause",
+      "licenseURL": "https://github.com/VIAME/VIAME"
```

Comment:
@Msgl could you update the `licenseURL` to point to a more explicit [URL](https://github.com/VIAME/VIAME/blob/master/LICENSE.txt) within the same repo. Thanks.